integration-test/src/main/resources/testsuites/e2e/gcp-longrunning-e2e-tests.yaml

```diff
@@ -7,5 +7,7 @@ tests:
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.freeipa.FreeIpaUpgradeTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxUpgradeDatabaseServerTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRepairTests
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxResizeTests
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxResizeRecoveryTests
         excludedMethods:
```

Comment:
Please remove the:
```
excludedMethods:
          - testSDXMediumDutyCreation
```
from here, the `testSDXMediumDutyCreation` method name is non existence in the project. So the whole exclude part can be removed from here.

**_Note:_**
_This was related to the_
```
- name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRepairTests
```
_instead of the_
```
- name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxResizeRecoveryTests
```